common/saber_base.h

```diff
@@ -42,6 +42,7 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(QUOTE)			\
     DEFINE_EFFECT(SECONDARY_IGNITION)		\
     DEFINE_EFFECT(SECONDARY_RETRACTION)		\
+    DEFINE_EFFECT(OFF)                          \
```

Comment:
Why do we need this.
We already have a way to trigger effects in InOutHelperTr

Reply:
This is for use in TrDoEffect<> to activate normal Off() instead of FAST_OFF

Reply:
A) It needs a better name.
B) It needs comments to explain what it's for.

Reply:
Actually, let's leave this for a later PR.
This thing will need support from the prop to work properly, right?
Let's add it when that support is added, that way I can check it and understand it properly.

Reply:
OK, removed.  Will add back with prop update and include comments.